src/d/actor/d_a_branch.cpp

```diff
@@ -52,7 +137,51 @@ void daBranch_Delete(daBranch_c*) {
 }
 
 /* 00000694-0000080C       .text daBranch_Create__FP10fopAc_ac_c */
-void daBranch_Create(fopAc_ac_c*) {
-    /* Nonmatching */
+s32 daBranch_Create(fopAc_ac_c* i_this) {
```

Comment:
if this is still nonmatching the comment should be preserved

Reply:
Added